ivadomed/utils.py

```diff
@@ -562,6 +586,40 @@ def structureWise_uncertainty(fname_lst, fname_hard, fname_unc_vox, fname_out):
     nib.save(nib_avgUnc, fname_avgUnc)
 
 
+def multiclass_dice_score(im1, im2):
+    im1 = np.asarray(im1[0, ]).astype(np.bool)
+    im2 = np.asarray(im2[0, ]).astype(np.bool)
```

Comment:
What are we extracting here? not sure to follow you re the first index.

Reply:
This shouldn't be here -> I modified it